### DISCUSSION TITLE
vlc Qt interface.

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#9950] [Q&A] [answered]
Now that vlc has been included into the main repo, I would like to get the qt interface activated/included into the build.  I have raised this before as an issue but it was just ignored. If this is not possible or doesn't make sense, please explain.

**Comment by xtkoba** [comment, marked as answer]
It is not ignored but remains unresolved due to lack of human resources. I appreciate your patience. Thanks.

**Reply by macerg** [reply]
Ok, thanks for explanation. Wish I could help. As a retired IBM sysprog I'd love to get involved but I just don't have the right skillset.  Let me know if you want a CICS system configured or some JCL written, hahaha.
